package exceptions;

public class ExceptionDemo1 {
	public static void main(String[] args) {
		int number = 20;
		System.out.println(1);
		
		try {
			System.out.println(number/0);
			System.out.println(3);
		}catch(Exception ex) {
			System.out.println(4);
			System.out.println("exception handled");
		}
		System.out.println(5);
	}

}
